h_transport_materials/property_database/pdag/palladium_silver.py

```diff
@@ -0,0 +1,149 @@
+    data_y=serra_diffusivity_data["deuteriumY"] * htm.ureg.m**2 * htm.ureg.s**-1,
+    isotope="D",
+    source="serra_hydrogen_1998-1",
+)
+
+serra_solubility_data = np.genfromtxt(
+    str(Path(__file__).parent) + "/serra_1998/solubility.csv",
+    delimiter=",",
+    names=True,
+)
+
+serra_solubility_h = Solubility(
+    units="m-3 Pa-1/2",
+    data_T=1000 / serra_solubility_data["hydrogenX"] * htm.ureg.K,
+    data_y=serra_solubility_data["hydrogenY"]
+    * htm.ureg.mol
+    * htm.ureg.m**-3
+    * htm.ureg.Pa**-0.5,
+    isotope="H",
+    source="serra_hydrogen_1998-1",
+)
+
+serra_solubility_d = Solubility(
+    units="m-3 Pa-1/2",
+    data_T=1000 / serra_solubility_data["deuteriumX"] * htm.ureg.K,
+    data_y=serra_solubility_data["deuteriumY"]
+    * htm.ureg.mol
+    * htm.ureg.m**-3
+    * htm.ureg.Pa**-0.5,
+    isotope="D",
+    source="serra_hydrogen_1998-1",
+)
+serra_dissociation_h = DissociationCoeff(
+    pre_exp=1.7e-2
+    * htm.ureg.mol
+    * htm.ureg.m**-2
+    * htm.ureg.s**-1
+    * htm.ureg.Pa**-1,
+    act_energy=26.294 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="H",
+    range=(323 * htm.ureg.K, 773* htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+)
+serra_dissociation_d = DissociationCoeff(
+    pre_exp=1.3e-2
+    * htm.ureg.mol
+    * htm.ureg.m**-2
+    * htm.ureg.s**-1
+    * htm.ureg.Pa**-1,
+    act_energy=24.780 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="D",
+    range=(323 * htm.ureg.K, 773* htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+)
+serra_recombination_coeff_h= RecombinationCoeff(
+    pre_exp=0.51 * htm.ureg.mol**-1 * htm.ureg.m**4 * htm.ureg.s**-1,
+    act_energy=65.490 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="H",
+    range=(323 * htm.ureg.K, 773 * htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+) 
+serra_recombination_coeff_d= RecombinationCoeff(
+    pre_exp=0.39 * htm.ureg.mol**-1 * htm.ureg.m**4 * htm.ureg.s**-1,
+    act_energy=61.842 * htm.ureg.kJ * htm.ureg.mol**-1,
+    isotope="D",
+    range=(323 * htm.ureg.K, 773 * htm.ureg.K),
+    source="serra_hydrogen_1998-1",
+) 
+vadrucci_permeability_wt84 = Permeability(
+    pre_exp=2.95e-8 * u.mol * u.m**-1 * u.s**-1 * u.Pa**-0.5,
+    act_energy=25.3149 * u.kJ * u.mol**-1,
+    range=(473 * u.K, 623 * u.K),
+    isotope="H",
+    source="vadrucci_pdag_2013",
+    note="wt = 84 micron"
+)
+vadrucci_permeability_wt150 = Permeability(
+    pre_exp=5.63e-8 * u.mol * u.m**-1 * u.s**-1 * u.Pa**-0.5,
+    act_energy=5.45606 * u.kJ * u.mol**-1,
+    range=(473 * u.K, 623 * u.K),
+    isotope="H",
+    source="vadrucci_pdag_2013",
+    note="wt = 150 micron"
+)
+vadrucci_permeability_wt200 = Permeability(
+    pre_exp=2.06e-8 * u.mol * u.m**-1 * u.s**-1 * u.Pa**-0.5,
+    act_energy=25.9256 * u.kJ * u.mol**-1,
+    range=(473 * u.K, 623 * u.K),
+    isotope="H",
+    source="vadrucci_pdag_2013",
+    note="wt = 200 micron"
+)
+vadrucci_dissociation_h = DissociationCoeff(
+    pre_exp=2.05e-3
```

Comment:
I can't find this value directly in the paper.
I believe that's because the authors give values for R_S and not k_1 directly.

Similarily, the activation energy of R_S is negative.

To avoid confusion, could we add our calculations details here?

Something like
```python
R_S_0 = 4.88e2 * htm.ureg.mol**-1 * htm.ureg.m**2 * htm.ureg.s * htm.ureg.Pa
E_R_S = -20.103 * htm.ureg.J * htm.ureg.mol**-1  # see comment below about this unit
vadrucci_dissociation_h = DissociationCoeff(
    pre_exp=2/R_S_0,
    act_energy=-E_R_S,
    isotope="H",
    range=(473* htm.ureg.K, 623* htm.ureg.K),
    source="vadrucci_pdag_2013",
    note="equation 17
)

```

Reply:
@FuerstT I directly committed the change